README.md

```diff
@@ -33,8 +33,27 @@ You can then run the Main object in Intellij as you can with any similar project
 
 You can also run `sbt universal:packageZipTarball` which creates a file `target/universal/tdr-consignment-export.tgz` which, when unzipped, gives you a `bin/tdr-consignment-export` executable which you can run with the same arguments as above. This is how the docker container runs the file so is the closest to running this in production.
 
+### Release notes
+We are now releasing new versions to github and for this we need release notes. 
+
+If no release notes are provided, the release job will generate them by running `git log $(git describe --tags --abbrev=0)..HEAD --oneline`
+
+You can write your own which will be used instead. 
+
+First, check what the next version will be. It will be the version in `version.sbt` withouth `-SNAPSHOT` on the end so version `0.0.1-SNAPSHOT` will become `0.0.1`
+
+Next, create a file in the `notes` directory called `${version}.markdown` and add in your release notes and commit and push these to the branch.
+
 ### Deployment
-Because this is run as an on demand ECS task, deployment is just pushing a new version of the docker image to ECR with the appropriate stage tag. This will be done by Jenkins as part of the deploy job on merge to master so there should be no reason to do this locally.
+There are multiple steps to the release process. On a merge to master:
```

Comment:
Which steps do you need to run to deploy to staging/prod?

Reply:
It's much the same as the others where you run the deploy job with the docker image version but this process creates two versions so I've put in a paragraph to describe that as well as how to deploy to other environments.